src/main/scala/org/allenai/plugins/archetypes/WebappPlugin.scala

```diff
@@ -23,6 +23,11 @@ object WebappPlugin extends AutoPlugin {
       val logNodeEnvironment = TaskKey[Unit](
         "logNodeEnvironment",
         "Logs the NodeJs build environment to SBT console")
+
+      val reStartWebapp = TaskKey[Unit](
+        "reStartWebapp",
+        "Restarts the service without running npm:build"
```

Comment:
Shouldn't this be something like: `"Restarts the service after rebuilding the front-end."`